mkosi/resources/man/mkosi.1.md

```diff
@@ -3232,6 +3232,11 @@ include <tunables/global>
 
   You can set `PORTABLE_PREFIXES=` in the `os-release` file yourself in a postinst script.
 
+- How can I make a tools tree that is not a directory?
+
+  The output format for tools trees cannot currently be changed via configuration files. Please use the `-t`
+  option on the commandline.
```

Comment:
If we add this we should also document how to build a non default tools tree in the first place

Reply:
I reworked this to add a tools tree section.